lib/membrane_element_mad/decoder.ex

```diff
@@ -18,15 +18,15 @@ defmodule Membrane.Element.Mad.Decoder do
   end
 
   @impl true
-  def handle_prepare(:stopped, state) do
+  def handle_prepare(:stopped, _, state) do
```

Comment:
In this case, an unnamed underscore is a bit confusing, I would change to `_ctx`